modules/client/examples/06-multisig-client/07-can-execute.ts

```diff
@@ -0,0 +1,27 @@
+/* MARKDOWN
+### Checking if user can approve in a multisig plugin
+*/
+import {
+  MultisigClient,
+  Context,
+  ContextPlugin,
+  ICanVoteParams,
+  CanExecuteParams,
+} from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an multisig client
+const client = new MultisigClient(contextPlugin);
+const canExecuteParams :CanExecuteParams = {
+  proposalId: "0x1234567890123456789012345678901234567890000000000000000000000001",
+  addressOrEns: "0x1234567890123456789012345678901234567890"
```

Comment:
Does this function require an address? If a proposal is passed, anybody should be able to execute it, only the process ID should be needed here

Reply:
Oh ok, I understood it more like  "can this address execute this proposalId", but I'll change it

Reply:
Just reviewed the contracts, it needs the `address` param